sql/parser/eval_test.go

```diff
@@ -250,8 +250,8 @@ func TestEvalExpr(t *testing.T) {
 		{`'2010-09-28'::timestamp`, `2010-09-28 00:00:00+00:00`},
 		{`('2010-09-28 12:00:00.1'::timestamp)::date`, `2010-09-28`},
 		{`'2010-09-28 12:00:00.1'::timestamp`, `2010-09-28 12:00:00.1+00:00`},
-		{`'2010-09-28 12:00:00.1+02:00'::timestamp`, `2010-09-28 10:00:00.1+00:00`},
-		{`'2010-09-28 12:00:00.1-07:00'::timestamp`, `2010-09-28 19:00:00.1+00:00`},
+		{`'2010-09-28 12:00:00.1+02:00'::timestamp`, `2010-09-28 12:00:00.1+02:00`},
```

Comment:
Furthermore, we need to convert values returning from the server to the session time zone on the client. This is currently not exactly possible because the session is opaque to the client.